samples/sensor/bmm350/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _BMM350:
+
+BMM350: Magnetometers
+########################################
+
+Description
+***********
+
+This sample application configures the Magnetometers to
```

Comment:
This sample shouldn't be needed, use the https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/sensor/magn_polling sample and an overlay instead

Or add a generic magn_trigger sample if the magn_polling sample does not fit the needs.

Reply:
I made the driver enable measuring by default so it is compatible with the magn_polling sample. Also, the extraneous sample was removed.